dataGraphs/thematics/CreativeWork/graphs/local-contexts-notice.json

```diff
@@ -0,0 +1,55 @@
+{
+    "@type": "CreativeWork",
+    "identifier": "{Notice ID}",
+    "name": [
+        {
+            "@language": "{language_tag}",
+            "@value": "{Notice Title}"
+        },
+        {
+            "@language": "{language_tag}",
+            "@value": "{Notice Title}"
+        }
+    ],
+    "author": {
+        "@type": "Organization",
+        "identifier": "https://oceanexpert.org/institution/xxxxxxx",
+        "legalName": "Local Contexts Inc.",
+        "name": "Local Contexts",
+        "url": "https://localcontexts.org/"
+    },
+    "text": [
+        {
+            "@language": "{language_tag}",
+            "@value": "{Notice Text}"
+        },
+        {
+            "@language": "{language_tag}",
+            "@value": "{Notice Text}"
+        }
+    ],
+    "url": "{NOTICE_PAGE_URL}",
+    "image": [
+        {
+            "@type": "ImageObject",
+            "url": "{ICON_IMG}",
+            "encodingFormat": "image/png"
+        },
+        {
+            "@type": "ImageObject",
+            "url": "{ICON_SVG}",
+            "encodingFormat": "image/svg"
+        }
+    ],
+    "dateCreated" : "{date Label created}",
+    "sdDatePublished": {"@type": "https://schema.org/Date"},
+    "sdLicense": [
+        {"@type": "https://schema.org/CreativeWork"},
+        {"@type": "https://schema.org/URL"}
+    ],
+    "sdPublisher": [
+        {"@type": "https://schema.org/Organization"},
+        {"@type": "https://schema.org/Person"}
+    ],
+    "creditText": "text: recommended citation"
```

Comment:
Not sure if this would go here but we have this text that we use "All Labels and Notices Icons and Template Texts © 2010 - 2024 Local Contexts. All Rights Reserved." (the year would have to be updated yearly or can we just have 2010 as the start and have it extend endlessly?

Reply:
@arojas1 that text sounds like a [copyrightNotice](https://schema.org/copyrightNotice), I have added it into this pull request.